go/tasks/plugins/k8s/spark/spark.go

```diff
@@ -284,7 +283,7 @@ func (sparkResourceHandler) BuildIdentityResource(ctx context.Context, taskCtx p
 func getEventInfoForSpark(sj *sparkOp.SparkApplication) (*pluginsCore.TaskInfo, error) {
 	state := sj.Status.AppState.State
 	isQueued := state == sparkOp.NewState ||
-		state == sparkOp.PendingSubmissionState ||
+		state == sparkOp.PendingRerunState ||
```

Comment:
Was this a new state added/ is the PendingSubmissionState deleted in v1beta2 ? Just wondering if this can be a breaking change and needs a corresponding bump in SparkOperator running in the cluster ?

Reply:
It looks to me like PendingSubmissionState does not exist in **v1beta1** too. 
 https://github.com/GoogleCloudPlatform/spark-on-k8s-operator/blob/537bab5683fcfe655d58366b76c2a6ed99a8e3f9/pkg/apis/sparkoperator.k8s.io/v1beta1/types.go#L260

It might require a corresponding bump in SparkOperator running in the cluster, but it depends on when you installed the spark-on-k8s-operator? PendingRerunState looks like it was added on Nov 19, 2018 (https://github.com/GoogleCloudPlatform/spark-on-k8s-operator/pull/291).  I'm not sure when exactly PendingSubmissionState was removed.

Reply:
Ah, now i remember. PendingSubmissionState comes from the multi-version support in https://github.com/GoogleCloudPlatform/spark-on-k8s-operator/blob/multi-version-support/pkg/apis/sparkoperator.k8s.io/v1beta2/types.go#L298 . 

I believe this might still be used by a few places . FYI @catalinii if this causes any issues

Reply:
@akhurana001 is it ok to resolve this conversation?  Are there any more concerns that need to be addressed?

Reply:
I think this is ok to be resolved.